wandb/cli/cli.py

```diff
@@ -2058,8 +2058,14 @@ def artifact():
     multiple=True,
     help="An alias to apply to this artifact",
 )
+@click.option("--id", "run_id", help="The run you want to upload to.")
+@click.option(
```

Comment:
Since this is untyped, it defaults to being a string (I think). We probably want to coerce this into a boolean if it's not None, and ensure that `False` and `FALSE` and `0` all get counted as "falsey". I'm not sure if we have or if click has a built-in way to do that, but I'd look for that first; it's possible that if you set the type to bool with a default of None it will treat it as Optional[bool] already.

Reply:
Setting `is_flag=True` inside the click declaration for `resume` should take care of what you're suggesting!

Reply:
Cool, I was hoping that was allowed (having a flag with a None default). Great!